app/controllers/rubcop_test_controller.rb

```diff
@@ -0,0 +1,17 @@
+class RubocopController  < ApplicationController
+	def initialize
+		print "this is so bad patern"
+	end
+
+	def print_fib(range)
+		range.each do |n|
+			p fib(n)
+		end
+	end
+
+	private
+		def fib(n)
```

Comment:
__[reek]__
* [UncommunicativeParameterName] has the parameter name 'n' [:ref](https://github.com/troessner/reek/blob/master/docs/Uncommunicative-Parameter-Name.md)

<!-- Comment by [SideCI](https://sideci.com) -->